tests/functional/api/groups/members_test.py

```diff
@@ -95,6 +102,109 @@ def test_it_returns_empty_list_if_no_members_in_group(self, app):
         assert res.json == []
 
 
+class TestListMembers:
```

Comment:
These are just the `TestListMembersLegacy` functests copy-pasted, then with the expected response bodies adjusted to the new format.